k4MarlinWrapper/src/components/MarlinProcessorWrapper.cpp

```diff
@@ -104,48 +104,34 @@ StatusCode MarlinProcessorWrapper::loadProcessorLibraries() const {
 
 
 std::shared_ptr<marlin::StringParameters> MarlinProcessorWrapper::parseParameters(
-  const Gaudi::Property<std::vector<std::string>>& parameters,
+  const Gaudi::Property<std::map<std::string, std::vector<std::string>>>& parameters,
   std::string& verbosity) const
 {
   auto parameters_ptr = std::make_shared<marlin::StringParameters>();
   info() << "Parameter values for: " << name() << " of type " << std::string(m_processorType) << endmsg;
-  std::string              parameterName   = "";
   std::vector<std::string> parameterValues = {};
```

Comment:
this can be moved inside the loop over parameters now.

Reply:
right, and then clearing the vector at the end of the loop ain't needed.